pkg/test/resourcefixture/testdata/basic/bigqueryreservation/v1alpha1/bigqueryreservationreservation/failovertononfailoverreservation/_http.log

```diff
@@ -94,7 +94,7 @@ X-Xss-Protection: 0
 
 ---
 
-PATCH https://bigqueryreservation.googleapis.com/v1/projects/${projectId}/locations/us-west2/reservations/bigqueryreservationreservation${uniqueId}?%24alt=json%3Benum-encoding%3Dint&updateMask=secondaryLocation
+PATCH https://bigqueryreservation.googleapis.com/v1/projects/${projectId}/locations/us-west2/reservations/bigqueryreservationreservation${uniqueId}?%24alt=json%3Benum-encoding%3Dint&updateMask=concurrency
```

Comment:
This PATCH request does not look right to me. The updateMask should include "secondaryLocation" to switch reservation to non-failover mode. (even if users do not change concurrency)